config/cola-elmo.conf

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Add a comment explaining what this file is for.

Reply:
You should have a note for everything that isn't obvious. Why this learning rate, for example?